### PR TITLE
Added several Fortinet device types

### PR DESCRIPTION
**Pull request by desnoe**

### CODE REVIEW

File: device-types/Fortinet/FG-2200E.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2200E
+slug: fg-2200e
+part_number: FG-2200E
+u_height: 2
+is_full_depth: false
```
Comment (danner26) [inline device-types/Fortinet/FG-2200E.yaml:7]: Please add `weight` if you have it

File: device-types/Fortinet/FG-2600F.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2600F
+slug: fg-2600f
+part_number: FG-2600F
+u_height: 2
+is_full_depth: true
```
Comment (danner26) [inline device-types/Fortinet/FG-2600F.yaml:7]: Please add `weight` if you have it

File: device-types/Fortinet/FG-300C.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 300C
+slug: fg-300c
+part_number: FG-300C
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Fortinet/FG-300C.yaml:7]: Please add `weight` if you have it

File: device-types/Fortinet/FG-60D.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 60D
+slug: fg-60d
+part_number: FG-60D
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Fortinet/FG-60D.yml:7]: Please add `weight` if you have it

Comment (danner26) [inline device-types/Fortinet/FG-60D.yml:7]: Please add `airflow` if you have it

File: device-types/Fortinet/FG-300C.yaml (hunk 3 above)
Comment (danner26) [inline device-types/Fortinet/FG-300C.yaml:7]: Please add `airflow` if you have it

File: device-types/Fortinet/FG-2600F.yaml (hunk 2 above)
Comment (danner26) [inline device-types/Fortinet/FG-2600F.yaml:7]: Please add `airflow` if you have it

File: device-types/Fortinet/FG-2200E.yaml (hunk 1 above)
Comment (danner26) [inline device-types/Fortinet/FG-2200E.yaml:7]: Please add `airflow` if you have it

File: device-types/Fortinet/FG-2200E.yaml [hunk 5]
```diff
@@ -0,0 +1,98 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2200E
+slug: fg-2200e
```
Comment (danner26) [inline device-types/Fortinet/FG-2200E.yaml:4]: Please append `fortinet-` to the slug

File: device-types/Fortinet/FG-2600F.yaml [hunk 6]
```diff
@@ -0,0 +1,102 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2600F
+slug: fg-2600f
```
Comment (danner26) [inline device-types/Fortinet/FG-2600F.yaml:4]: Please append `fortinet-` to the slug

File: device-types/Fortinet/FG-300C.yaml [hunk 7]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 300C
+slug: fg-300c
```
Comment (danner26) [inline device-types/Fortinet/FG-300C.yaml:4]: Please append `fortinet-` to the slug

File: device-types/Fortinet/FG-60D.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 60D
+slug: fg-60d
```
Comment (danner26) [inline device-types/Fortinet/FG-60D.yml:4]: Please append `fortinet-` to the slug

File: device-types/Fortinet/FG-60D.yml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+part_number: FG-60D
+u_height: 1
```
Comment (danner26) [inline device-types/Fortinet/FG-60D.yml:6]: It looks like this is not rack mountable, according to the doc for the `60D` the device is desktop or wall-mount. Please make this `u_height: 0`

File: device-types/Fortinet/FG-2200E.yaml [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+part_number: FG-2200E
+u_height: 2
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
```
Comment (danner26) [inline device-types/Fortinet/FG-2200E.yaml:11]: Are these modular/hot swappable? If so, you should remove the `power-ports` definitions, add `module-bays` definition and create the applicable PSUs in the module-types folder if they do not already exist

File: device-types/Fortinet/FG-2600F.yaml [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+part_number: FG-2600F
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
```
Comment (danner26) [inline device-types/Fortinet/FG-2600F.yaml:11]: Are these modular/hot swappable? If so, you should remove the `power-ports` definitions, add `module-bays` definition and create the applicable PSUs in the module-types folder if they do not already exist

File: device-types/Fortinet/FG-60D.yml [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: nema-5-15p
```
Comment (danner26) [inline device-types/Fortinet/FG-60D.yml:13]: It looks like the `60D` is a `dc-terminal` not a `nema-5-15p`. Please adjust this accordingly

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Fortinet/FG-60D.yml (hunk 4 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml:7]: It is fanless.

Comment (danner26) [reply device-types/Fortinet/FG-60D.yml:7]: So it has `passive` airflow then

**Comment by danner26** [comment]
I am going to make this PR a draft again, when it is ready to be re-reviewed please mark it as ready for review. Thanks!

File: device-types/Fortinet/FG-60D.yml (hunk 8 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml:4]: most Fortinet devices don't append it, and all Fortigate are without it, so I prefer to keep it this way.

File: device-types/Fortinet/FG-60D.yml (hunk 12 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml:13]: you're right, I'll fix it

File: device-types/Fortinet/FG-2200E.yaml (hunk 1 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2200E.yaml:7]: will add

File: device-types/Fortinet/FG-2600F.yaml (hunk 2 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2600F.yaml:7]: will add

File: device-types/Fortinet/FG-300C.yaml (hunk 3 above)
Comment (desnoe) [reply device-types/Fortinet/FG-300C.yaml:7]: will add

File: device-types/Fortinet/FG-60D.yml (hunk 4 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml:7]: will add

File: device-types/Fortinet/FG-60D.yml (hunk 9 above)
Comment (danner26) [reply device-types/Fortinet/FG-60D.yml:6]: I will confirm with my team how we want to handle this, but this is my general thinking:
If a device is not rack mountable according to the manufacturer, i.e. the manufacturer only lists the devices as `Desktop and wall-mountable` but does not sell a rack mount adapter, then the device should be listed as 0U. During the creation of the device, the user can then change it to 1U if the device does indeed utilize a 3rd part adapter to make it rack mountable

File: device-types/Fortinet/FG-2200E.yaml (hunk 10 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2200E.yaml:11]: Yes, there are hot swappable, but after I can't find the information on how to model them correctly: I'm missing the model name and the part number (at least one of them). I'll remove the power ports in the device types.

File: device-types/Fortinet/FG-2600F.yaml (hunk 11 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2600F.yaml:11]: Yes, there are hot swappable, but after I can't find the information on how to model them correctly: I'm missing the model name and the part number (at least one of them). I'll remove the power ports in the device types.

Comment (danner26) [reply device-types/Fortinet/FG-2600F.yaml]: A good example of how to model the PSU module type is: (you need the model/part number though_

```
---
manufacturer: Cisco
model: UCS-PSU-6332-AC
part_number: UCS-PSU-6332-AC
power-ports:
  - name: PSU{module}
    type: iec-60320-c14
    maximum_draw: 650
```

File: device-types/Fortinet/FG-2200E.yaml (hunk 10 above)
Comment (danner26) [reply device-types/Fortinet/FG-2200E.yaml]: A good example of how to model the PSU module type is: (you need the model/part number though_

```
---
manufacturer: Cisco
model: UCS-PSU-6332-AC
part_number: UCS-PSU-6332-AC
power-ports:
  - name: PSU{module}
    type: iec-60320-c14
    maximum_draw: 650
```

File: device-types/Fortinet/FG-60D.yml (hunk 9 above)
Comment (danner26) [reply device-types/Fortinet/FG-60D.yml:6]: Please leave this unresolved until the maintainers come to a consensus. Thank you.

File: device-types/Fortinet/FG-60D.yml (hunk 4 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml:7]: good to know, I'll fix it

File: device-types/Fortinet/FG-300C.yaml (hunk 3 above)
Comment (desnoe) [reply device-types/Fortinet/FG-300C.yaml:7]: Can't find this info

Comment (desnoe) [reply device-types/Fortinet/FG-300C.yaml:7]: Well after looking at the photo it must be front-to-rear. I'll add this info.

File: device-types/Fortinet/FG-2600F.yaml (hunk 2 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2600F.yaml:7]: I'll add this info.

File: device-types/Fortinet/FG-2200E.yaml (hunk 1 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2200E.yaml:7]: I'll add this info.

File: device-types/Fortinet/FG-2200E.yaml (hunk 5 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2200E.yaml:4]: I'll do it

File: device-types/Fortinet/FG-2600F.yaml (hunk 6 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2600F.yaml]: I'll do it

File: device-types/Fortinet/FG-300C.yaml (hunk 7 above)
Comment (desnoe) [reply device-types/Fortinet/FG-300C.yaml]: I'll do it

File: device-types/Fortinet/FG-60D.yml (hunk 8 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml]: Thanks for this clarification. I'll do it.

File: device-types/Fortinet/FG-2200E.yaml (hunk 10 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2200E.yaml]: I know how to do it but I'm missing the model and part number.

File: device-types/Fortinet/FG-2600F.yaml (hunk 11 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2600F.yaml]: I know how to to it but I'm missing the model & part number.

**Comment by danner26** [comment]
Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
```
weight: 12.01
weight_unit: lb
```

Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

**Comment by danner26** [comment]
Hey @desnoe please ping me when this PR is ready for review. Thank you

**Comment by desnoe** [comment]
@danner26, just rebased the branch, please go on.

File: device-types/Fortinet/FG-2600F.yaml [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+manufacturer: Fortinet
+model: FortiGate 2600F
+slug: fortinet-fg-2600f
+part_number: FG-2600F
+u_height: 2
+is_full_depth: true
+weight: 13.9
+weight_unit: kg
+airflow: front-to-rear
+console-ports:
```
Comment (danner26) [inline device-types/Fortinet/FG-2600F.yaml:11]: Not sure what happened to the power, can you want to add those back as module bays?

File: device-types/Fortinet/FG-2200E.yaml [hunk 14]
```diff
@@ -0,0 +1,92 @@
+---
+manufacturer: Fortinet
+model: FortiGate 2200E
+slug: fortinet-fg-2200e
+part_number: FG-2200E
+u_height: 2
+is_full_depth: false
+weight: 18.2
+weight_unit: kg
+airflow: front-to-rear
```
Comment (danner26) [inline device-types/Fortinet/FG-2200E.yaml:10]: Not sure what happened to the power, can you want to add those back as module bays?

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Fortinet/FG-60D.yml (hunk 9 above)
Comment (danner26) [reply device-types/Fortinet/FG-60D.yml:6]: So here is what we are going with. The device itself is not rack mountable. Thus in this repo, which is suppose to be a source of truth, this should have `u_height: 0`. Then, if you specifically want to use this as a rack mountable with the third party adapter, you can manually edit the device definition in your NetBox instance.

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Fortinet/FG-60D.yml (hunk 9 above)
Comment (desnoe) [reply device-types/Fortinet/FG-60D.yml]: done

File: device-types/Fortinet/FG-2600F.yaml (hunk 13 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2600F.yaml:11]: done

File: device-types/Fortinet/FG-2200E.yaml (hunk 14 above)
Comment (desnoe) [reply device-types/Fortinet/FG-2200E.yaml:10]: done

**Comment by desnoe** [comment]
@danner26 this should be better now, could you have a look at it please ?

**Review by danner26 — APPROVED** [review]

**Pull request merged**